containers/deploy/pulp/deployments.yml

```diff
@@ -575,3 +573,119 @@
   tags:
     - start
     - restart
+
+- name: Stop squid containers by scaling replicas down to 0
```

Comment:
Can this look more like the tasks from #4 ?

Reply:
Yea -- I'll merge yours first then update and rebase mine.